PersistentStore/PersistentStore.cpp

```diff
@@ -495,6 +504,7 @@ namespace WPEFramework {
 
             bool success = false;
 
+            SQLITE_LOCK
```

Comment:
Can we avoid app level locks for the reads? This would restrict simultaneous reads from the PersistentStore.